core-bundle/composer.json

```diff
@@ -72,6 +72,7 @@
         "friendsofsymfony/http-cache": "^2.15.1",
         "friendsofsymfony/http-cache-bundle": "^2.6",
         "guzzlehttp/promises": "^1.5",
+        "masterminds/html5": "^2.8",
```

Comment:
We need this for best compatibility with HTML5. We'll be able to drop it as soon as we require PHP 8.4 [which will bring native support for DOM spec compatibility](https://wiki.php.net/rfc/domdocument_html5_parser) 🥳